samples: profiler: Update documentation (#2380)
nrfconnect/sdk-nrf: samples/profiler/README.rst

=======

After programming the sample to your board, you can test it by running the script ``real_time_plot.py`` (located under :file:`scripts/profiler`).

> As an argument, pass the test preset name (for example, run ``real_time_plot.py test_name``).

Reviewer: How do I know what name I should pass?

Author: You can pass any name here. It will be used later for naming the files with stored data.
For example if you call pass: `test123`, the files will be named `test123.csv` and `test123.json`

Author: I can change it to `pass the arbitrary test preset name`. Will this be ok ?

> As an argument, pass the test preset name (for example, run ``real_time_plot.py test_name``).
> The test preset name is used to generate names for csv and json files, that are used to store the data.

Reviewer: ```suggestion
As an argument, pass the name that should be used to store the data.
For example, run ``real_time_plot.py test_name`` to generate a :file:`test_name.csv` and a :file:`test_name.json` file.
```

Author: Thank you. I will squash the change.
